src/tools/kotlin-generation-utils.ts

```diff
@@ -0,0 +1,84 @@
+/**
+ * @license
+ * Copyright (c) 2020 Google Inc. All rights reserved.
+ * This code may only be used under the BSD style license found at
+ * http://polymer.github.io/LICENSE.txt
+ * Code distributed by Google as part of this project is also
+ * subject to an additional IP rights grant found at
+ * http://polymer.github.io/PATENTS.txt
+ */
+
+/**
+ * Kotlin language formatting preferences.
+ */
+export interface KotlinPreferences {
+  indent: number;
+  lineLength: number;
+}
+
+/**
+ * Default language formatting settings.
+ */
+export const KT_DEFAULT: KotlinPreferences = {indent: 4, lineLength: 120};
```

Comment:
100, not 120

Reply:
Although a line length limit seems like it might be overkill for generated code. Particularly for a code generator implementation that is still temporary and will go away once me move to KotlinPoet

Reply:
OK actually the implementation is really nice. I like it!

Reply:
Thanks, I always confuse which style guide we use.